Maui.RevenueCat.InAppBilling/Enums/OwnershipId.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Maui.RevenueCat.InAppBilling.Enums;
+public enum OwnershipId
+{
+    FamilyShared,
+    Purchased,
+    Unknown,
+}
```

Comment:
Oh I see, I have started this `Id` naming convention. But I think it is incorrect.

When I would see `OwnershipId` in the code I would think that it is a simple number/guid and not an enum.
Rename it to `OwnershipType` (yes there will be a problem with references, but you can specify the RevenueCat reference explicitly with a whole/partial path).

- on the other hand we would have something like this `entitlement.OwnershipType.ToOwnershipType(),` what is quite confusing. Fck, any ideas how to handle this ?

- if we decide to rename it then please change also `ToOwnershipId` on both Android and iOS

Reply:
I wasn't too fond of the Id naming convention either, but trying to be consistent...  I also would have gone with OwnershipType.  I'm going to make it happen.  

I think the way to handle entitlement.OwnershipType.ToOwnershipType()  is to call it entitlement.OwnershipType.ToOwnershipTypeFromNative() . It's still not super but I think it give enough of a clue as to what is going on to not be confusing.

I also fixed StoreId and PeriodId with similar changes.

Reply:
I didn't resolve as this is significant enough to warrant a rereview.

Reply:
I don't know if it is just me, but when you go fast throught the name it gives me an impression that it is mapping to native type. Because usually in mapping methods you mention the type you are **mapping to** and not the one you are **mapping from**.

What about smth like: `ToAbstractOwnershipType` ?
that way we would know it is mapping to abstracted type ?

Reply:
Maybe ToDtoOwnershipType since you are already using Dto in the names of a lot of them?  Abstract to me has a very different meaning due to it being a keyword.

Reply:
yes, this one makes sense. 
After you add this I will approve and finally merge the PR. 
Thank you so much for the contribution

Reply:
Done